src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -69,6 +71,7 @@ public class RecordService {
   static final String CONTENT = "content";
   static final String META = "meta";
   static final String SCHEMA_ID = "schema_id";
+  static final String CONNECTOR_PUSH_TIME = "ConnectorPushTime";
```

Comment:
we have used all lower case historically
to be consistent, may be use `connector_push_time`?

Reply:
nvm, I see `CreateTime`

Reply:
should we just have `PushTime` (or flush) time to make it consistent with `CreateTime`?

Reply:
Imo `PushTime` sounds quite ambiguous. It isn't clear where exactly the push happened. Ideally I'd have `SnowflakeConnectorPushTime`, but it's too long and I reckon in most cases it should be obvious for users what connector is implied. 

I'd like not to use "flush", as we still have an ingest-sdk that buffers data, so it isn't a time when a real flush to Snowflake happens. Also, after removing a buffering layer in the connector, nothing is going to be flushed as records will be passed to ingest-sdk immediately.

Reply:
I vote for `SnowflakeConnectorPushTime` too. 

TBH, it should be `SnowflakeConnectorPullTime` to be very specific. (Kafka connect polling so w.r.t snowflake it is pull time)

i am fine with either given we will and should be very descriptive in our docs.

Reply:
I would even go with adding `Ms` in suffix

Reply:
I'm avoiding pull for pull-kafka reasons. 

I'm not sure if ms is necessary as the other createtime doesn;t have it and it's also millisecond value?

Reply:
1. After reconsideration, naming the field as `SnowflakeConnectorPushTime` makes more sense, as customers can use some source connector, so it isn't clear which one's timestamp it is.
1. I wouldn't go with _ms_ suffix in order to align with the Kafka's naming for timestamp: `CreateTime`, `LogAppendTime`.